src/lib/create-from-script-tag.js

```diff
@@ -32,10 +66,23 @@ function createFromScriptTag(createFunction, scriptTag) {
 
   form.insertBefore(container, scriptTag);
 
-  createFunction({
+  createOptions = {
     authorization: authorization,
     container: container
-  }).then(function (instance) {
+  };
+
+  WHITELISTED_DATA_ATTRIBUTES.forEach(function (compositeKey) {
+    var value = scriptTag.getAttribute('data-' + compositeKey);
+
+    if (value == null) {
+      return;
+    }
+
+    addCompositeKeyValuePairToObject(createOptions, compositeKey, value);
+  });
+
+  createFunction(createOptions).then(function (instance) {
+    analytics.sendEvent(instance._client, 'integration-type.script-tag');
```

Comment:
Not super pleased with grabbing the private prop `_client` here, but this was the easiest way to include an analytic.